src/components/pages/pages.pw.tsx

```diff
@@ -0,0 +1,194 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+
+import { PagesProps } from "../pages";
+import {
+  dataComponentButtonByText,
+  backArrow,
+} from "../../../playwright/components/pages";
+import {
+  getComponent,
+  getDataElementByValue,
+} from "../../../playwright/components";
+import { CHARACTERS } from "../../../playwright/support/constants";
+import {
+  checkAccessibility,
+  waitForAnimationEnd,
+} from "../../../playwright/support/helper";
+import {
+  PageComponent,
+  PageComponentWithTitle,
+  PagesComponent,
+} from "./components.test-pw";
+
+test.describe("Prop checks for Pages component", () => {
+  ["initialPageIndex", "pageIndex"].forEach((propName) => {
```

Comment:
**suggestion** (non-blocking): I'm not sure of what the "right" thing to do is, and I know this structure was just copied from the Cypress tests - but personally I'm not a fan of the nested `describe`s, with each being in a `forEach` loop. Since there are only 2 prop names being tested I think I'd actually prefer to just repeat the inner block twice, once for each of the 2 prop names we want to test.

But it's not something I'm going to hold this up over - just a thought.